cel/cos_tlv.go

```diff
@@ -1,31 +1,51 @@
 package cel
 
-import "crypto"
+import (
+	"crypto"
+	"fmt"
+)
 
 const (
-	// CosTypeValue indicate the CELR event is a COS content
+	// CosEventType indicate the CELR event is a COS content
 	// TODO: the value needs to be reserved in the CEL spec
-	CosTypeValue uint8 = 80
+	CosEventType uint8 = 80
+)
+
+// Type for COS nested events
+const (
+	ImageRefType      uint8 = iota
+	ImageDigestType   uint8 = iota
```

Comment:
nit: here and below, no need for `= iota`

Reply:
done